utils/main_api_class.py

```
@@ -2,8 +2,9 @@
 
 import requests
 import pandas as pd
-from . enums import *
-from .connector_utils import install_libraries
+import flatten_json
+from utils.enums import *
```

Ruff (0.8.2)

`from utils.enums import *` used; unable to detect undefined names

(F403)